lib/cn/node_split.c

```diff
@@ -559,8 +559,13 @@ cn_split(struct cn_compaction_work *w)
     /* Poll for all our kvset-split work to complete.  Ideally, we'd call
      * flush_workqueue() here, but that can severely stall new requests.
      */
-    while (inflight > 0)
-        usleep(100 * 1000);
+    while (inflight > 0) {
+        const struct timespec req = {
+            .tv_nsec = 100 * 1000
+        };
+
+        hse_nanosleep(&req, NULL, "nodsplit");
```

Comment:
nodesplit

Reply:
Too long, we only show the first 8 chars in the "wait channel" field...

Reply:
We could extend that, but traditionally it's only 6 or 8 chars wide in conventional use...

Reply:
That is fine then

Reply:
Lemme change it to "nodesplt", then I'll update other wait messages to use "splt" rather than "split".  That way you get fewer hits when grepping for it!

Reply:
$ grep -R splt lib
lib/cn/cn_tree.c:            cv_wait(&node->tn_ss_cv, &node->tn_ss_lock, "spltwait");
lib/cn/node_split.c:        hse_nanosleep(&req, NULL, "nodesplt");

FTW!